osquery/events/kernel/circular_queue_user.cpp

```diff
@@ -100,3 +102,5 @@ int CQueue::kernelSync() {
   return update.drops;
 }
 
+}  // namespace osquery
+
```

Comment:
nit, ditch this last blank line after the namespace close, just end with a newline.